src/cpp/core/include/core/text/CsvParser.hpp

```diff
@@ -138,6 +138,19 @@ std::pair<std::vector<std::string>, InputIterator> parseCsvLine(
          std::vector<std::string>(), begin);
 }
 
+template <typename InputIterator>
+InputIterator parseCsvLine(InputIterator begin,
```

Comment:
I added this overload primarily because the main version that returns a `std::pair<>` can often be cumbersome to work with.